packages/manager/modules/iam/src/components/policyAdd/policyAdd.controller.js

```diff
@@ -0,0 +1,9 @@
+import { PATTERN_POLICY } from '@iam/constants';
+
+export default class PolicyAddController {
+  /* @ngInject */
+  constructor() {
+    this.policy = { name: '' };
+    this.PATTERN_POLICY = PATTERN_POLICY;
```

Comment:
This line can be removed, all the constants are injected onto the $rootScope

Reply:
It's fixed.